site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc

```diff
@@ -0,0 +1,45 @@
+[api]
+ifeval::["{base_field_name}" != "identityProviderLink"]
+[field]#identityProviderLinks# [type]#[Array]#::
+The list of Link objects.
+endif::[]
+
+[field]#{base_field_name}.displayName# [type]#[String]#::
+The display name for the user.
+
+[field]#{base_field_name}.identityProviderId# [type]#[UUID]#::
+The unique Id of the identity provider.
+
+[field]#{base_field_name}.identityProviderUserId# [type]#[String]#::
+The Id for the User that is provided by the identity provider.
+
+[field]#{base_field_name}.insertInstant# [type]#[Long]#::
+The link:/docs/v1/tech/reference/data-types/#instants[instant] that the Tenant was added to the FusionAuth database.
+
+[field]#{base_field_name}.lastLoginInstant# [type]#[Long]#::
+The link:/docs/v1/tech/reference/data-types/#instants[instant] when the User logged in last.
+
+[field]#{base_field_name}.tenantId# [type]#[UUID]#::
+The Id of the Tenant that this User belongs to
+
+[field]#{base_field_name}.token# [type]#[String]#::
+The access token, this string is an encoded JSON Web Token (JWT).
```

Comment:
This may or may not be an access token. In general we hope it is a refresh token, but we don't necessary know and it will vary by IdP. 

If it is an access token, it may or may not be a JWT, and if it is a refresh token, it likely is not a JWT. In all cases, we will treat it as opaque. 

Also, this field is now where tokens are stored, previously they were stored in `registration.tokens` map. We could add a note here that indicates this is where they should be finding the token in 1.28.0 and beyond.

Reply:
@robotdan (and @mooreds for funzies)
I may or may not copy/paste this comment as the field definition here.  😆 If I do paste it, it is likely confusing for most people, but may be not confusing for some people.

Reply:
I love cutting and pasting @robotdan's comments into documentation. It's like a brainwave transfer that benefits everyone :) .

Reply:
Not sure where the best place to document this is, but since it varies by IdP do we also want to add some indication of where one might look to find out what kind of "token" this is?  Like: "To see what type of token is stored for this IdP see [here]".